### DISCUSSION TITLE
What repo to start with ?

### DISCUSSION THREAD

**Discussion by parvinsingh** [bluesky-social/atproto#3290] [General Discussion]
Under this link- https://github.com/bluesky-social 

There are lot of repos. There is an example stratusphere repo. However, if I need to start and fork one of the repos to build my own private bskySocial kind of social network, that is invite only. Which repo do I start with ? 
As a starting point - if I can run it locally and open the webpage locally that will be great.

thank you !!

**Comment by X1Vi** [comment]
I am working with social-app one it is related to the frontend mostly and is quite easy to start with. Good Luck 👍 
Use tools from vscode to navigate the project easily.
